chef/cookbooks/pacemaker/spec/providers/primitive_spec.rb

```diff
@@ -33,11 +33,13 @@ def cib_object_class
 
     it "should modify the primitive if it has different params" do
       expected_configure_cmd_args = [
-        %'--set-parameter "os_password" --parameter-value "newpasswd"',
-        %'--delete-parameter "os_tenant_name"',
+        %'--set-parameter os_password ' +
+          '--parameter-value \$ne\!w\%pa\\\'ss\&wo\"rd',
+        %'--delete-parameter os_tenant_name',
       ].map { |args| "crm_resource --resource #{fixture.name} #{args}" }
       test_modify(expected_configure_cmd_args) do
-        new_params = Hash[fixture.params].merge("os_password" => "newpasswd")
+        new_params = Hash[fixture.params].merge(
+          "os_password" => "$ne!w%pa'ss&wo\"rd")
```

Comment:
Would it be possible to use `%{$ne!w%pa'ss&wo"rd}` instead of `"$ne!w%pa'ss&wo\"rd"`? It'd make it more visible that it's `"` and not `\"` :-)